WindsorBody/Scripts/python-files/IOFoamLib.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
```

Comment:
This needs some short documentation what it is, what it does, and how to use it.